ui/src/app/views/cohort-edit/component.html

```diff
@@ -1,9 +1,12 @@
 <h2>Set Cohort Info</h2>
-<div *ngIf="user; else loggedout">
 <div class="form-area">
   <div>
-    <label >Cohort Name:</label>
-    <input type="text" id="name" class="input" name="cohort-name" [(ngModel)]="cohort.name" autofocus>
+    <label>Cohort Name:</label>
+    <input type="text" id="name" class="input" name="cohort-name" autofocus
+        [(ngModel)]="cohort.name" (ngModelChange)="autoGenerateId()">
```

Comment:
I think we only want to generate an ID once, because the concept of the ID is immutable, correct? (I know we wanted them to be based off the name, but I think that this is incompatible with the idea of immutable IDs?)